app/views/hub/documents/_form.html.erb

```diff
@@ -17,6 +17,7 @@
       <%= f.vita_min_text_field :display_name, t('hub.documents.display_name'), classes: ["form-width--long"] %>
       <%= f.cfa_select :document_type, t("general.document_type"), DocumentTypes::ALL_TYPES.map { |doc_type| [doc_type.label, doc_type.key] }, include_blank: true %>
       <%= f.cfa_select :tax_return_id, t("general.tax_return"), @client.tax_returns.map { |tax_return| [tax_return.year, tax_return.id] }, include_blank: true %>
+      <%= f.cfa_checkbox :archived, t(".archived_label") %>
```

Comment:
Can you swap this out with f.hub_checkbox so that it matches the checkbox style in the mock?